user/src/main/java/com/scaler/user/models/User.java

```diff
@@ -1,26 +1,36 @@
 package com.scaler.user.models;
 
+import com.scaler.user.utilities.RoleType;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 import java.sql.Timestamp;
-import java.util.List;
 
 @Getter
 @Setter
+@AllArgsConstructor
 @NoArgsConstructor
+@Entity
+@Table(name = "Users")
 public class User {
+
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Id
     private Long userId;
     private String fullName;
+    @Column(unique = true)
     private String username;
-    private String password;
+    @Column(unique = true)
     private String email;
```

Comment:
Both columns will have duplicate data so, hold only one column value and delete the other one.